app/views/payments/_form.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= form_with(model: payment) do |form| %>
+  <% if payment.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(payment.errors.count, "error") %> prohibited this payment from being saved:</h2>
+
+      <ul>
+        <% payment.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="container">
+
+  <div class="form-group">
+    <%= form.label :Name, class: "col-sm-2 col-form-label" %>
+    <div class="col-sm-10">
+      <%= form.text_field :name, class: "form-control" %>
+    </div>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :Amount, class: "col-sm-2 col-form-label" %>
+    <div class="col-sm-10">
+      <%= form.number_field :amount, class: "form-control" %>
+    </div>
+  </div>
+  <div class="form-group">
+    <%= form.label :Category_Id, class: "col-sm-2 col-form-label" %>
+    <div class="col-sm-10">
+      <%= form.number_field :category_id, class: "form-control" %>
+    </div>
```

Comment:
- [ ] It will be better if you add a dropdown list, so the user can choose which category will add to it because the user does not know what you mean by the id and if adding an id which not exists will raise an error.